pandas/tests/frame/indexing/test_indexing.py

```diff
@@ -1526,3 +1526,18 @@ def test_loc_iloc_setitem_non_categorical_rhs(
         # "c" not part of the categories
         with pytest.raises(TypeError, match=msg1):
             indexer(df)[key] = ["c", "c"]
+
+    @pytest.mark.parametrize("indexer", [tm.getitem, tm.loc, tm.iloc])
```

Comment:
`indexer_sli`

Reply:
What is "sli"?

Reply:
(and to be explicit: all the tests above use `indexer` as keyword name)

Reply:
> What is "sli"?

sli  -> setitem, loc, iloc

> (and to be explicit: all the tests above use indexer as keyword name)

probably predates the fixture.  ill edit on the next "assorted cleanups" branch

Reply:
> sli -> setitem, loc, iloc

This test is about `getitem` (I know getitem and setitem are exactly the same under the hood, but it doesn't make it clearer to read the code ..)

> probably predates the fixture. ill edit on the next "assorted cleanups" branch

I would argue to not do that. The current way here is much more explicit and readable, for very little extra code (and not even an extra _line_ of code)

Reply:
OK